src/components/masters/program/main-content/data/service.js

```diff
@@ -1,9 +1,9 @@
 import apiClient from '../../../../../apiClient';
 
-const fetchProgramEnrollmentOverview = (programUUID) => {
+const fetchProgramEnrollment = (programUUID) => {
```

Comment:
`fetchProgramCourseEnrollments`

Reply:
renamed to fetchProgramCourseEnrollments done!